src/plugins/query_enhancements/common/types.ts

```diff
@@ -36,3 +36,36 @@ export interface QueryStatusOptions<T> {
 }
 
 export type FetchFunction<T, P = void> = (params?: P) => Promise<T>;
+
+export interface SQLQueryResponse {
+  status: string;
+  schema: Array<{ name: string; type: string }>;
+  datarows: Array<Array<string | null>>;
```

Comment:
should it be
```suggestion
  datarows: Array<Array<unknown>>;
```

i think columns in SQL response is not always string?

Reply:
[53198c4](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8609/commits/53198c40ecdcea449979f3eb51eee7782300cb54)